examples/twittermap/web/public/javascripts/common/services.js

```diff
@@ -434,6 +435,38 @@ angular.module('cloudberry.common', ['cloudberry.mapresultcache'])
             // unrecognized map type
             break;
         }
+      },
+
+      pinMapOneTweetQuery: function(pinid){
+
+          var setpinfilter = [{
+              field: "id",
+              relation: "in",
+              values: pinid
+          }];
+
+          //sql
+          var pinDBquery = (JSON.stringify({
+              dataset:"twitter.ds_tweet",
+              filter: setpinfilter,
+              select:{
+                  order: ["-create_at"],
+                  limit: defaultSamplingSize,
+                  offset: 0,
+                  field: ["id","text","user.id","create_at","user.name","user.profile_image_url"]
+              },
+              transform:{
+                  wrap:{
+                      id:"pinMapOneTweetResult",
+                      category:"pinMapOneTweetResult"
+                  }
+              }
+          }));
+
+          //changes the id's type from string to int64.
+          pinDBquery = pinDBquery.replace(/"(\d+)"/,"[$1]");
```

Comment:
This just removes double quotes?

Reply:
Yep, I tried many ways and this one works fine. Plus, we need to make sure it is int so we must remove double quotes.